packages/mdc-slider/constants.js

```diff
@@ -20,6 +20,9 @@ const strings = {
   TRACK_SELECTOR: '.mdc-slider__track',
   TRACK_FILL_SELECTOR: '.mdc-slider__track-fill',
   THUMB_SELECTOR: '.mdc-slider__thumb',
+  TRACK: 'mdc-slider__track',
```

Comment:
Put TRACK, TRACK_FILL and SLIDER in `cssClasses` instead of `strings`

Reply:
Done!